python/setup.py

```diff
@@ -13,23 +13,21 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
-
```

Comment:
Is this still used to run our examples? If so we might want to clean this up and look at the code that's still being used

Reply:
as discussed we'll be creating a follow up task.